conda/recipes/dask-cudf/meta.yaml

```diff
@@ -43,6 +43,7 @@ requirements:
   run:
     - python
     - cudf ={{ version }}
+    - pynvml >=11.4.1,<12.0.0a0
```

Comment:
Pushed this small change to include `pynvml` in the `dask-cudf` Conda package. Hope that is ok 🙂